lms/djangoapps/mobile_api/urls.py

```diff
@@ -12,6 +12,7 @@
     url(r'^my_user_info', my_user_info),
     url(r'^video_outlines/', include('mobile_api.video_outlines.urls')),
     url(r'^course_info/', include('mobile_api.course_info.urls')),
+    url(r'^discussion_course/', include('discussion_api.urls')),
```

Comment:
We don't want to add a new endpoint to the mobile_api.  The mobile api code should be able to access the discussion endpoints without actually adding them here.